be/src/vec/sink/vdata_stream_sender.h

```diff
@@ -117,14 +117,22 @@ class Channel {
                         (_brpc_dest_addr.port == config::brpc_port)),
               _serializer(_parent, _is_local) {}
 
+#ifdef BE_TEST
```

Comment:
Do not use another constructor. Just pass a `TNetworkAddress` to determine `_is_local` is true or false.